app/uk/gov/hmrc/servicedependencies/config/ServiceDependenciesConfig.scala

```diff
@@ -19,46 +19,34 @@ package uk.gov.hmrc.servicedependencies.config
 import java.io.File
 import java.nio.file.Path
 
-import play.api.Play
-import play.api.libs.json.Json
-import uk.gov.hmrc.play.config.ServicesConfig
-import uk.gov.hmrc.servicedependencies.config.model.CuratedDependencyConfig
+import com.google.inject.{Inject, Singleton}
+import play.api.{Configuration, Play}
+import uk.gov.hmrc.play.bootstrap.config.BaseUrl
 
 import scala.concurrent.duration._
 import scala.io.Source
 
-trait CacheConfig {
-  def cacheDuration: FiniteDuration
-}
 
-trait ReleasesConfig {
-  def releasesServiceUrl: String
-}
 
-class ServiceDependenciesConfig(configPath: String) extends CacheConfig with ReleasesConfig with ServicesConfig {
+@Singleton
+class ServiceDependenciesConfig @Inject()(appConfiguration: Configuration) extends BaseUrl  {
```

Comment:
You should be able to `override val configuration: Configuration` instead of using appConfiguration and overriding configuration

Reply:
👍